exporter/kafkaexporter/marshaler_test.go

```diff
@@ -56,6 +58,7 @@ func TestDefaultMetricsMarshalers(t *testing.T) {
 func TestDefaultLogsMarshalers(t *testing.T) {
 	expectedEncodings := []string{
 		"otlp_proto",
+		"otlp_json",
```

Comment:
I believe it would be worth adding a test demonstrating what's the JSON message that is expected to be exported for a specific payload.

Reply:
If you think there's a test that covers this already, just let me know.

Reply:
Sorry I took my time doing so, I have added a test for tracing data.